lib/dal-materialized-views/src/incoming_connections.rs

```diff
@@ -150,7 +144,8 @@ async fn prop_to_prop(ctx: &DalContext, component_id: ComponentId) -> Result<Vec
                         AttributeValue::path_from_root(ctx, from_attribute_value_id).await?;
                     let from_component_id =
                         AttributeValue::component_id(ctx, from_attribute_value_id).await?;
-                    let from_prop_id = AttributeValue::prop_id(ctx, attribute_value_id).await?;
```

Comment:
was this just an oopsie wrong variable? or .. am i missing something more subtle here?

Reply:
I was just about to comment. This was the second bug!